connexion/decorators/json_validator.py

```diff
@@ -0,0 +1,62 @@
+from jsonschema import Draft4Validator, _utils
```

Comment:
This isn't a decorator, IMHO it should be up one directory. What do you think?

Reply:
What do you think about renaming jsonref.py (only contains one function) to jsonschema.py and adding it there?

Reply:
sounds good to me